### PR TITLE
[Doc] Fix Lint errors

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
Signed-off-by: EsoragotoSpirit <wanglichen@starrocks.com>

## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Review by amber-create — DISMISSED** [review]

File: docs/integrations/superset_support.md [hunk 1]
```diff
@@ -51,8 +57,9 @@ df = pd.read_sql(sql, conn)
 ```
 
 ### Superset Example
-In superset, use `Other` database, and set url is：
-```
+
+In superset, use `Other` database, and set url is:
```
Comment (evelynzhaojie) [inline docs/integrations/superset_support.md:61]: is 是 as 吗？

File: docs/integrations/datadog_support.md [hunk 2]
```diff
@@ -44,27 +55,31 @@ Uninstall
 ### Configuration
 
 #### Configuration FE
```
Comment (evelynzhaojie) [inline docs/integrations/datadog_support.md:57]: FE configuration, 下面是 BE configuration? 改一下基线

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Comment by EsoragotoSpirit** [comment]
run starrocks_admit_test

**Comment by EsoragotoSpirit** [comment]
run starrocks_be_unittest

**Pull request merged**